views/includes/navigation.ejs

```diff
@@ -0,0 +1,8 @@
+<header class="main-header">
+    <nav class="main-header__nav">
+        <ul class="main-header__item-list">
+            <li class="main-header__item"><a class="<%= path === '/' ? 'active' : '' %>" href="/">Shop</a></li>
+            <li class="main-header__item"><a class="<%= path === '/admin/add-product' ? 'active' : '' %>" href="/admin/add-product">Add Product</a></li>
```

Comment:
can we shift these type of path validations to some other steps? like the initial stage of when we process req.

Reply:
No, We need the implement it on Html , so that User can see the active page .